package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rimraf ./build",
     "lint": "eslint ./statistics/**/* ./lib/**/* ./spec/**/*",
     "format": "npm run lint -- --fix",
-    "test": "nyc ./node_modules/jasmine/bin/jasmine.js",
+    "test": "nyc jest --coverage",
```

Comment:
Please refer this [comment](https://github.com/reportportal/agent-js-jasmine/pull/91#pullrequestreview-2084560198). Let's collect coverage via jest only.

Reply:
done